server/repositories/entities/field.ts

```diff
@@ -17,6 +17,6 @@ export default class Field {
   @Column({ nullable: true })
   validation: string
 
-  @OneToMany('Field', 'id')
-  field: Array<Field> | null
+  @OneToOne('Field', 'id')
+  field: Field
```

Comment:
Hmm, I hought OneToMany was right? Also I missed it first time round but shouldn't this be FieldValue? And is it actually needed? Would the application ever want to get all possible FieldValues for a given field 🤔

Reply:
I'm confusing myself now (not difficult); let me have another look.

Reply:
Yeah, this was completely unnecessary